data/relay.json

```diff
@@ -11361,7 +11361,7 @@
               "defaultValue": null
             },
             {
-              "name": "information",
+              "name": "embed_data",
```

Comment:
embed_data:  hold overridden fields (i.e. {title: 'new title'})
embed: our method to return embed object (annotation)